packages/ipfs-cli/src/commands/init.js

```diff
@@ -69,22 +70,20 @@ module.exports = {
     const IPFS = require('ipfs-core')
     const Repo = require('ipfs-repo')
 
-    const node = await IPFS.create({
-      repo: new Repo(repoPath),
-      init: false,
-      start: false,
-      config
-    })
-
     try {
-      await node.init({
-        algorithm: argv.algorithm,
-        bits: argv.bits,
-        privateKey: argv.privateKey,
-        emptyRepo: argv.emptyRepo,
-        profiles: argv.profile,
-        pass: argv.pass,
-        log: print
+      await IPFS.create({
+        repo: new Repo(repoPath),
+        init: {
+          algorithm: argv.algorithm,
+          bits: argv.bits,
+          privateKey: argv.privateKey,
+          emptyRepo: argv.emptyRepo,
+          profiles: argv.profile,
+          pass: argv.pass
```

Comment:
Ah good catch!

Reply:
Ok I went over the code base and it looks like nothing is ever using this `log` property & I think I probably removed it because TS told me it was not used.